app/containers/HeaderButton/Common.tsx

```diff
@@ -6,15 +6,15 @@ import Container from './HeaderButtonContainer';
 import Item from './HeaderButtonItem';
 
 interface IHeaderButtonCommon {
-	navigation: any;
-	onPress?(): void;
+	navigation: any; // TODO: Evaluate proper type
```

Comment:
This probably will end up being too complex, we might need to take a look later on this one. Something close to this:

```suggestion
	navigation: CompositeNavigationProp<
				DrawerNavigationProp<
					DrawerParamList,
					| 'AdminPanelStackNavigator'
					| 'ChatsStackNavigator'
					| 'DisplayPrefStackNavigator'
					| 'ProfileStackNavigator'
					| 'SettingsStackNavigator'
				>,
				StackNavigationProp<ModalStackParamList>
		  >
```